features/step_definitions/dns.rb

```diff
@@ -0,0 +1,20 @@
+Then(/^the hostname "([^"]+)" should resolve to "([^"]+)" on the internal DNS server$/) do |host, ip|
```

Comment:
I like the overall idea of cucumber, but I am not so sure about these steps. They rely on Unix tools like  _dig_ and _killall_. This won't work on Windows. I spend quite a some effort to a) get Landrush to work on Windows and b) make it possible to develop on Windows including running the test suite. This would break this again.

If we can find a way to make this cucumber tests work w/o relying on Unix tools I am all up for it. For example instead of using _dig_ one could just use Ruby's Resolv:DNS class.